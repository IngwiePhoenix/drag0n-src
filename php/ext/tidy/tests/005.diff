001+ zend_mm_heap corrupted
001- <html>
002- <head>
003- <title></title>
004- </head>
005- <body>
006- </body>
007- </html>