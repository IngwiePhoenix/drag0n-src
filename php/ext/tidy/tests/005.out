zend_mm_heap corrupted